README.md

```diff
@@ -44,6 +44,8 @@ Use the following environment variables to set additional configuration options:
  - `SAUCE_BUILD` - the text that will be displayed as Build Name on SauceLabs.
 
  - `SAUCE_CONFIG_PATH` - path to a file which contains additional **job options** as JSON. See [SauceLabs Test Configuration](https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-TestAnnotation) for a full list.
+
+ - `SAUCE_CONNECT_OVERRIDES_PATH` = path to a file which contains overrides for the the connector. See [Sauce Connect Launcher Advanced Usage](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for details.
```

Comment:
@titerman, could you please check this description? I suggest something like the following:
```suggestion
 - `SAUCE_CONNECT_OVERRIDES_PATH` - path to a file which contains overrides for the Sauce Connect options to be passed to the [SauceLabs connector](https://github.com/DevExpress/saucelabs-connector). See [Sauce Connect Launcher Advanced Usage](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for details about these options.
```

Reply:
I went ahead and made the update. Let me know if there is anything else I can do. Happy to help.